lib/src/main/java/bio/terra/javapfb/Library.java

```diff
@@ -0,0 +1,45 @@
+package bio.terra.javapfb;
+
+import bio.terra.common.logging.LoggingInitializer;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.fasterxml.jackson.module.paramnames.ParameterNamesModule;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.builder.SpringApplicationBuilder;
+import org.springframework.boot.context.properties.ConfigurationPropertiesScan;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+import org.springframework.retry.annotation.EnableRetry;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+@SpringBootApplication(
+    scanBasePackages = {
+      // Scan for logging-related components & configs
+      "bio.terra.common.logging",
```

Comment:
Just trying to follow all the pieces - are these `bio.terra.common` libraries used?  Also don't understand this whole `/lib` directory - is it leftover boilerplate or is it necessary?

Reply:
RE: terra common lib references - those are boiler plate, I can probably remove them.

RE: lib directory: Here's my thoughts on the general code organization: 

- lib module: Contains code to perform operations on PFB files
- java-pfb-cli module: contains CLI-related code and will reference the pfb client
- client module: Builds the client from the lib code

Reply:
Update on code structure:
library module: Contains code to perform operations on PFB files
cli module: contains CLI-related code and will reference the pfb client